docs/ref/forms/modelform-meta.txt

```diff
@@ -0,0 +1,283 @@
+==========================
+ModelForm ``Meta`` options
+==========================
+
+.. module:: django.forms.models
+    :synopsis: ModelForm inner Meta class options
+
+
+.. admonition:: Other ModelForm-related documentation
+
+    This document enumerates and describes the options that can be used to
+    structure a :class:`~django.forms.ModelForm` through its inner ``Meta``
+    class. For additional context, examples and use cases for the
+    :class:`~django.forms.ModelForm` class, please refer to the documentation
+    topic :doc:`Creating forms from models </topics/forms/modelforms>` and the
+    reference entry for :doc:`ModelForm functions</ref/forms/models>`.
```

Comment:
We won't need it after we've moved the docs to their rightful place in ref/forms/models.txt as it is already provided